sample_apps/run_example.sh

```diff
@@ -1108,7 +1108,7 @@ function get_measurement_of_trusted_simple_app_under_islet() {
     # get_measurement_of_app_by_name "islet_example_app.exe"
 
     run_cmd
-    local mrenclave="6190EB90B293886C172EC644DAFB7E33EE2CEA6541ABE15300D96380DF525BF9"
```

Comment:
Suggestion: Given that we have come across an instance where this hash-ID had to be changed (for whatever reasons), can you please add a comment before L1111 documenting the reason why this changed and why it had to be updated.

If you can add a comment capturing the reasons why this changed in this instance, and put down some notes alerting future readers that this kind of [recurring] update may be needed in the future, it will help us guide to this area while troubleshooting future such failures.

The main message would be to document "how-to" figure out the new hash-ID string. (I, myself, would like to know how you figured out how-to get this new hash-ID string.)

Reply:
I updated comments. Thanks for the suggestion!